### ISSUE TITLE
[UI improvement] display webcam modes in a better way

### ISSUE THREAD

**Issue by yiids** [ossia/score#1466]
When you add a webcam, you just a list of all the supported modes and resolution. This should be broken down into some drop down menus for easy selecting.
![image](https://user-images.githubusercontent.com/62114515/227711981-8d1e2a6f-05f6-4bea-8450-739bb2bffb62.png)

**Comment by yiids** [comment]
They seem to have disapeared completely from the input list, now i only see a device called "Streams" 
![image](https://github.com/ossia/score/assets/62114515/69908dbb-ba62-432f-92d7-84f611469ad9)

**Comment by jcelerier** [comment]
yes, in 3.1.11 an underlying ffmpeg update broke the detection of V4L2 cameras ; they are back on master: https://github.com/ossia/score/releases/tag/continuous

**Comment by jcelerier** [comment]
how is the current UI for you @yiids?

**Comment by yiids** [comment]
I think it would be better if all the different resolutions and modes would be in drop down menus in the settings part of the add device dialog

**Comment by HugoGenillier** [comment]
I made a Figma mockup for this issue. Tell me if it works well, i think this fits what you said @yiids 
![cam](https://github.com/ossia/score/assets/118723648/49bac15f-1530-4021-94be-5b742530a6cc)
(The style of the dropdown will be the same as the other ossia score's dropdown, this is just a quick mockup)

**Comment by jcelerier** [comment]
Ideally we would have a first dropdown with the video mode (YUYV, RGB, etc) and a second with the resolution

**Comment by yiids** [comment]
> Ideally we would have a first dropdown with the video mode (YUYV, RGB, etc) and a second with the resolution

this ^ for the rest it was exactly what i had in mind @HugoGenillier
